### ISSUE TITLE
Failing corePDEs_SteadyHeatConstrainedOpt2D_Dirichlet_Dist_Param_Tpetra test in weaver and CEE intel build 

### ISSUE THREAD

**Issue by ikalash** [sandialabs/Albany#886]
Looks like the problem NaNs out: 

```
    Convergence Tolerance: 1e-07
  With #Eqns=1681, #RHSs=1 ...

p=2: *** Caught standard std::exception of type 'Thyra::CatastrophicSolveFailure' :

 /projects/albany/nightlyAlbanyCDash/repos/Trilinos/packages/stratimikos/adapters/belos/src/Thyra_BelosLinearOpWithSolve_def.hpp:700:
 
 Throw number = 2
 
 Throw test that evaluated to true: true
 
 /projects/albany/nightlyAlbanyCDash/repos/Trilinos/packages/belos/src/BelosStatusTestImpResNorm.hpp:635:
 
 Throw number = 1
 
 Throw test that evaluated to true: true
 
 Belos::StatusTestImpResNorm::checkStatus(): One or more of the current implicit residual norms is NaN.
```

https://sems-cdash-son.sandia.gov/cdash/test/3203189
https://sems-cdash-son.sandia.gov/cdash/test/3203122

@mperego, @kliegeois : can you please have a look or tag the best person to look at this?

**Comment by mperego** [comment]
@ikalash there are a couple of tests that randomly fail and I haven't been able to reproduce the iusse yet. I have access to cee.compute020, can you make the albany builds readable?

**Comment by ikalash** [comment]
They should be readable already.  I just ran ```chmod ug+rwX -R /projects/albany/nightlyAlbanyCDash/*; chgrp wg-albany -R /scratch/albany/*; chmod ug+rwX -R /scratch/albany/*```.  Maybe try again now?  I noticed myself that this appears to be a heisenbug that shows up in some builds some nights but not others.  Very strange...

**Comment by ikalash** [comment]
My cronjob actually changes permissions to group ones exactly for this reason (so others can access the folders).  But I saw something that may not have been quite right.  Can you please try again now?  I was changing to group permissions of for all the subdirs of /scratch/albany but not /scratch/albany itself.

**Comment by mperego** [comment]
I can access now. Thanks!

**Comment by ikalash** [comment]
Excellent!  I'll go ahead and fix the permission changes in all the cronjobs on CEE.

**Comment by mperego** [comment]
Hopefully this is related to issue #873, which should have been fixed by PR #899.
